src/controllers/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './handlers/http'
+export * from './handlers/uws'
+// export * from './handlers/rpc'
```

Comment:
This was left commented, but I think that it might be better if neither them exported here, to avoid any confusion between the HTTP and RPC controllers. What do you think about it?
Also, we could export the handlers for the indexes of each protocol so the imports are less verbose.